workers/ohsome_quality_analyst/api/api.py

```diff
@@ -337,8 +348,36 @@ async def metadata_topic_by_key(key: TopicEnum) -> TopicResponse:
     return TopicResponse(result=get_topic_definition(key.value))
 
 
+@app.get(
+    "/metadata/indicators",
+    tags=["metadata"],
+    response_model_exclude={
+        "result": {"__all__": {"label_description": True, "result_description": True}}
+    },
+)
```

Comment:
Could not get this to work using exclude config in pydantic models. Also this only excludes the fields in the actual response but not the model (not in the openAPI specs).

Reply:
Ideally the metadata class would not hold the label and result description attributes.